stack.yaml

```diff
@@ -57,6 +57,7 @@ extra-deps:
 - recover-rtti-0.4.0.0@sha256:2ce1e031ec0e34d736fa45f0149bbd55026f614939dc90ffd14a9c5d24093ff4,4423
 - lock-file-0.7.0.0@sha256:3ad84b5e454145e1d928063b56abb96db24a99a21b493989520e58fa0ab37b00,4484
 - http-client-0.7.11
+- terminal-size-0.3.3
```

Comment:
```suggestion
# lts 18.28 provides 0.3.2.1 but we need at least 0.3.3
- terminal-size-0.3.3
```

Reply:
we wanna add explanations for all extra-deps so we can know when they're redundant